Delete preinst

nuehm-arno opened the pull request.

olof-nord commented:
This script is removed because it isnt used anymore? Why was it added, and why is it removed? Asking to try to understand more about deb packaging.

nuehm-arno commented:
I added them some time ago, because I wanted to make sure, that Axolotl is closed before upgrading/removing. But while deeper testing, it came clear, that if Debian helper scripts are closed with `exit 1`, the upgrading/removing fails.
And the case of `exit 1` occurs, when there is no Axolotl binary to be closed.
So I removed them from packaging and started to find a solution, but couldn't find one. So these two helpers are useless now...
